spec/integ/megolm-integ.spec.ts

```diff
@@ -30,17 +41,21 @@ const ROOM_ID = "!room:id";
  * @param {TestClient} recipientTestClient
  * @return {Promise} promise for Olm.Session
  */
-function createOlmSession(olmAccount, recipientTestClient) {
-    return recipientTestClient.awaitOneTimeKeyUpload().then((keys) => {
-        const otkId = Object.keys(keys)[0];
-        const otk = keys[otkId];
-
-        const session = new global.Olm.Session();
-        session.create_outbound(
-            olmAccount, recipientTestClient.getDeviceKey(), otk.key,
-        );
-        return session;
-    });
+async function createOlmSession(olmAccount: Olm.Account, recipientTestClient: TestClient): Promise<Olm.Session> {
```

Comment:
imho we should remove the params doc-block if it doesn't provide any additional info or update it according to the types.
In this case `@return {Promise<Olm.Session>}`